spec/models/comment_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  it 'is invalid without a user'
```

Comment:
Incomplete model tests here. You'll want to test for the association. Take another look at the article I sent you for reference.